src/button-split.scss

```diff
@@ -76,4 +76,16 @@ $fd-button-split-margin: 0.125rem / 2;
       }
     }
   }
+
+  &__text {
```

Comment:
Can't you target the first child instead of adding an additional modifier class?

Reply:
But text is not wrapped in any element, and you cannot really target a `TextNode`.